airflow/www/views.py

```diff
@@ -3041,7 +3044,13 @@ class ConnectionModelView(wwwutils.SuperUserMixin, AirflowModelView):
         'extra__google_cloud_platform__key_path': StringField('Keyfile Path'),
         'extra__google_cloud_platform__keyfile_dict': PasswordField('Keyfile JSON'),
         'extra__google_cloud_platform__scope': StringField('Scopes (comma separated)'),
-        'extra__google_cloud_platform__num_retries': IntegerField('Number of Retries'),
+        'extra__google_cloud_platform__num_retries': IntegerField(
+            'Number of Retries',
+            validators=[
+                validators.Optional(strip_whitespace=True),
+                validators.NumberRange(min=0),
+            ],
+        ),
         'extra__grpc__auth_type': StringField('Grpc Auth Type'),
         'extra__grpc__credentials_pem_file': StringField('Credential Keyfile Path'),
```

Comment:
Does this need to change too ?

```diff
- 'extra__grpc__credentials_pem_file': StringField('Credential Keyfile Path'),
+ 'extra__grpc__credential_pem_file': StringField('Credential Keyfile Path'),
```

Reply:
Ah. yes or the other side should be `extra__grpc__credentials_pem_file`.